pkg/util/iptables/iptables_test.go

```diff
@@ -1109,8 +1109,15 @@ func TestRestoreAllGrabOldLock(t *testing.T) {
 	runner := newInternal(&fexec, ProtocolIPv4, TestLockfilePath)
 	defer os.Remove(TestLockfilePath)
 
+	socketName := "@xtables"
+	_, err := os.Stat(socketName)
+	if err == nil { // socket exists, need to delete it
```

Comment:
I'm not sure about this... what does this do if multiple instances of the test are running in parallel?

Reply:
I thought the test were running serially.. Did you know if we could specify a different socket name each time?

Reply:
> I thought the test were running serially..

I think they each run in their own process, but could run in parallel

> Did you know if we could specify a different socket name each time?

It looks hard-coded in the iptables.go file, so I don't think so